rust/xaynet-metrics-macro/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "xaynet-metrics-macro"
+version = "0.1.0"
+authors = ["Xayn Engineering <engineering@xaynet.dev>"]
+edition = "2018"
+
+[dependencies]
+syn = "1.0.37"
+quote = "1.0"
+
+[lib]
+proc-macro = true
+
+[features]
+metrics = []
```

Comment:
I'm curious about how this works. Are feature flags passed to the dependencies? Specifically, if we compile `xaynet` with the `metrics` feature flag, then cargo will also build `xaynet-metrics-macro` with the `metrics` feature? That doesn't seem right to me.